### PR TITLE
Adding docs for deploy containers

### PR DESCRIPTION
**Pull request by aowen14**
Adding documentation for deploying containers on Shadeform. Uses a notebook from examples which leverages an lm-eval docker container to run mistral through a benchmark using cost-effective A6000's.

### CODE REVIEW

File: deployinggpucontainers.mdx [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+### Setup
+
+The requirements are simple, so in your preferred python environment:
+
+```bash
+git clone https://github.com/shadeform/examples.git
+cd examples/
+```
+Then in `deploy_container.ipynb` input your Shadeform API Key from [here](https://platform.shadeform.ai/settings/api), and a  `huggingface_token` if the model you need is gated (like Llama2 or Gemma).
```
Comment (cholical) [inline deployinggpucontainers.mdx]: We don't support a huggingface api token on the settings api page

Comment (cholical) [reply deployinggpucontainers.mdx]: The customer will have to pass it into their docker container using the environment variable supported by the specific container

File: deployinggpucontainers.mdx [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+The notebook kicks off a VM request and runs the container specifically on a 1xA6000 Machine which are quite afforable on our platform. The container will download the model and run the benchmark for us. 
+
+The notebook leverages a lot of the principles found in [this post](https://docs.shadeform.ai/findingmostaffordablegpus) for using the platform to _find_ affordable GPU's.
```
Comment (cholical) [inline deployinggpucontainers.mdx]: nit: can we change "this post" to "this guide"

File: deployinggpucontainers.mdx [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  "cloud": best_instance["cloud"],
+  "region": region,
+  "shade_instance_type": shade_instance_type,
+  "shade_cloud": True,
+  "name": "cool_gpu_server",
+  "launch_configuration": {
+    "type": "docker",
+    "docker_configuration": {
+      "image": "shadeform/lm-eval-harness",
+      "envs": [
```
Comment (cholical) [inline deployinggpucontainers.mdx:47]: Can we add a line or two describing that this is passing environment variables into the container?

File: deployinggpucontainers.mdx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+```bash
+ssh -i <location-of-your-pem-file> shadeform@<ip-address-copied-from-cell-output>
+```
+
+Once in, we can check the logs via the container id. The `-a` flag is if it has already finished.
+```bash
+docker ps -a
+docker container logs <container-id>
+```
+
```
Comment (cholical) [inline deployinggpucontainers.mdx:99]: Can we add a paragraph here interpreting the results of the eval? For example, "From the results of the evaluation, we can see that the Mistral model ...."

File: deployinggpucontainers.mdx (hunk 1 above)
Comment (aowen14) [reply deployinggpucontainers.mdx]: Okay, will edit for clarity. What I mean is that into the notebook, they will need to put 2 different things: 1. The api key from Shadeform, that they can get on your page, and 2 a hugging face token iff they need to use a llama or gemma model that need authorized access.

Comment (aowen14) [reply deployinggpucontainers.mdx]: The container launch cell in the notebook shows this

File: deployinggpucontainers.mdx (hunk 2 above)
Comment (aowen14) [reply deployinggpucontainers.mdx]: yes, will do

File: deployinggpucontainers.mdx (hunk 3 above)
Comment (aowen14) [reply deployinggpucontainers.mdx:47]: yes, will also put this into the notebook source as well

File: deployinggpucontainers.mdx (hunk 4 above)
Comment (aowen14) [reply deployinggpucontainers.mdx:99]: yes, will do, left that commentary out for simplicity

**Pull request merged**